packages/fetchai/connections/p2p_libp2p/connection.py

```diff
@@ -34,13 +34,18 @@
 from aea.connections.base import Connection, ConnectionStates
 from aea.crypto.base import Crypto
 from aea.exceptions import enforce
+from aea.helpers.acn.acn_message_pb2 import AcnMessage, AeaEnvelope
+from aea.helpers.acn.acn_message_pb2 import AgentRecord as AgentRecordPb
+from aea.helpers.acn.acn_message_pb2 import Status
```

Comment:
No :(. Please move that into a "protocols" package: packages/fetchai/protocols/acn and give it version 0.1.0. We will then upgrade it to the proper protocol which is already on the other branch when we have merged it to develop!

Reply:
moved locally to each package, will introduce protocol in a final version, see no reason to create another protocol